components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/ClaimMetadataManagementService.java

```diff
@@ -151,4 +151,14 @@ void removeExternalClaim(String externalClaimDialectURI, String externalClaimURI
     default void removeClaimMappingAttributes(int tenantId, String userstoreDomain) throws ClaimMetadataException {
 
     }
+
+    /**
+     * Remove all claims of a given tenant.
+     *
+     * @param tenantId Id of the tenant
+     * @throws ClaimMetadataException
```

Comment:
Complete the javadoc comment.

Reply:
Check every other instance and fix this.

Reply:
Fixed in #3312